openshift/07-route.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: "cors-proxy-route"
+objects:
+- apiVersion: v1
+  kind: Route
+  metadata:
+    name: cors-proxy
+    labels:
+      app: cors-proxy
+  spec:
+    host: ${ENVIRONMENT}cors-proxy.${WILDCARD_DOMAIN}
```

Comment:
Wouldn't be better to leave this empty and let openshift to figure it out? OpenShift can know its public domain and create unique host per project/route.

Reply:
yes but our clusters right know does not have this issue (https://github.com/3scale/infrastructure/issues/252) solved yet. So, right now, openshift would create a local route which you cant edit the hostname and we would need to create a new route by hand.

I would leave this as it is today and once we have the infra issue solved we could rollback and leave the host creation to Openshift.

Another solution would bet the entire host as a parameter...

What do you think?

Reply:
I'd go with the whole host as a param.

Reply:
Ok!